File: JavaScript solutions/1447. Simplified Fractions.js
/**
 * @param {number} n
 * @return {string[]}
 */
var simplifiedFractions = function (n) {
  const gcd = (a, b) => {
    if (a == 0) return b;
    return gcd(b % a, a);
  };

  const ans = [];
  for (let i = 1; i < n; i++) {
    for (let j = i + 1; j <= n; j++) {
      if (gcd(i, j) == 1) ans.push(`${i}/${j}`);
    }
  }
  return ans;
};

File: C++ solutions/1447. Simplified Fractions.cpp
class Solution {
public:
    int gcd(int a, int b) {
        if (a == 0) return b;
        return gcd(b % a, a); 
    }
    vector<string> simplifiedFractions(int n) {
        vector<string>ans;
        for(int i = 1; i < n; i++) {
            for(int j = i + 1; j <= n; j++) {
                if (gcd(i, j) == 1)
                    ans.push_back(to_string(i) + "/" + to_string(j));
            }
        }
        return ans;
    }
};